_data/simple-icons.json

```diff
@@ -235,6 +245,11 @@
             "hex": "FF5722",
             "source": "https://www.blogger.com"
         },
+        {
+            "title": "Boost",
+            "hex": "F68313",
+            "source": "https://en.wikipedia.org/wiki/File:Boost_Mobile_logo.svg"
```

Comment:
Their homepage actually provides an SVG version of their logo, so it would be better to specify that as source. Also the color they're using there is `#F69220` rather then `#F68313`

Reply:
I'll update this.